Proto/cg.bat

```diff
@@ -0,0 +1,7 @@
+@echo off
```

Comment:
I really believe we should stop with the batch files in build process.

By nature they aren't suitable for cross platform support, and also, they are relatively painful to debug / step through compared to a .fsx script.

I've had first attempt at build.fsx using FAKE and was able to have first steps of building proto and dotnet40 working on windows + first steps for the same for mac support, I really think we should go this route.

In the meantime, keeping all things in the root build.cmd will make it easier for migration if we could agree it would help in future.